### PR TITLE
chore: move docs out of  `sdk` crate folder

### PR DESCRIPTION
**Pull request by antbern**
## Motivation and Context

To improve the layout of the project, move the docs outside the `sdk` crate folder 💯 

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [x] Code follows project style and guidelines.
- [x] No redundant or duplicate code.
- [x] No added new dependencies without clear justification and approval.

### Documentation

- [x] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [x] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [x] Tests pass locally.

### Build & CI/CD

- [x] Confirmed no CI/CD pipeline issues.
- [x] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

**Review by hauju — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+```bash
+cd bindings/wasm
+wasm-pack build --dev --scope eto
+```
+
+Finally you can build and serve the docs:
+
+```bash
+mkdocs serve
+```
```
Comment (kristifidani) [inline docs/README.md:23]: One must say that this is all what needs to be done and then complain why is there no rust or wasm docs. 😆  
Can we add a small explanation that those are not built automatically, and they need to build it manually and move it inside this directory? Something like that 😄

Comment (antbern) [reply docs/README.md:39]: Done 💯
